pkg/gui/refresh.go

```diff
@@ -264,6 +265,7 @@ func (gui *Gui) refreshRebaseCommits() error {
 		return err
 	}
 	gui.State.Model.Commits = updatedCommits
+	gui.State.Model.WorkingTreeState = gui.git.Status.WorkingTreeState()
```

Comment:
If we're now setting this on the model, we should update the places that use `gui.git.Status.WorkingTreeState()` to now use `gui.State.Model.WorkingTreeState`, for the sake of consistency. But if we do that, I wonder how often that value would be stale. Perhaps never because it doesn't change without the list of commits also changing?

Reply:
Hm, I'm not sure I agree; the state in the model is really only there for making sure that the state matches the commits when drawing the commit list. For all other callers, I see no reason to use the model value; if we base behavior decisions on it (e.g. which options to show in the rebase menu), then that shouldn't depend on how long ago we last refreshed the commits list; it's better to use the most up to date value in these cases.

Maybe we could rename the model state to something like `WorkingTreeStateCorrespondingToCommits` to make that clearer.

Reply:
Makes sense. How about `WorkingTreeStateAtLastCommitRefresh`?

Reply:
I like that; force-pushed.